language_names/manual.py

```diff
@@ -0,0 +1,32 @@
+import sqlite3
+import textwrap
+import csv
+
+
+def populate_database():
+    conn = sqlite3.connect('test.db')
+    c = conn.cursor()
+    c.execute(textwrap.dedent("""
+        PRAGMA foreign_keys=OFF;"""))
+    c.execute(textwrap.dedent("""
+        BEGIN TRANSACTION;"""))
+    c.execute(textwrap.dedent("""
+        CREATE TABLE IF NOT EXISTS fixes (id integer primary key, lg text, inLg text, name text, unique(lg, inLg) on conflict replace);
```

Comment:
Suggest each field on its own line here and below. Also, ON CONFLICT REPLACE should be capital as should field types, e.g. INTEGER PRIMARY KEY.

Reply:
Okay I'll make these changes